hw2/03-count.html

```diff
@@ -6,28 +6,38 @@
     <title>03 - Count</title>
     <link
       rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
     <script defer src="03-count.js"></script>
   </head>
-  <body>
+  <body class="bg-primary">
     <main>
-      <h1>Exercise 03 - Count</h1>
-      <hr />
-      <div>
-        <label>Highlight all occurences of a word:</label>
-        <input />
-      </div>
-      <hr />
-      <div>
-        It was the best of times, it was the worst of times, it was the age of
-        wisdom, it was the age of foolishness, it was the epoch of belief, it
-        was the epoch of incredulity, it was the season of Light, it was the
-        season of Darkness, it was the spring of hope, it was the winter of
-        despair, we had everything before us, we had nothing before us, we were
-        all going direct to Heaven, we were all going direct the other way—in
-        short, the period was so far like the present period, that some of its
-        noisiest authorities insisted on its being received, for good or for
-        evil, in the superlative degree of comparison only.
+      <div class="container">
+        <div class="row mt-5 justify-content-center">
+          <div class="col-8 card p-3">
```

Comment:
Great use of the card class. I didn't think to use that and I believe I used container instead